test/bats/cli-test.bats

```diff
@@ -34,6 +34,14 @@ load helpers
     assert_cmd_verify_failure
 }
 
+@test "notation verifier crl test" {
```

Comment:
this looks a negative test, can you update the description. And which existing one is for positive test? Can you add a comment to that test as well?

Reply:
Sure thing.